src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -213,6 +233,8 @@ public void testBottom() throws Exception {
             createModuleConfig(ImportOrderCheck.class);
         checkConfig.addAttribute("option", "bottom");
         final String[] expected = {
+            "8: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "java.io.IOException"),
+            "11: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "javax.swing.JComponent"),
```

Comment:
there should be violation on 14 and 17 lines too.
If separator should not between groups - all blank lines in imports should be violated.

Reply:
Right,my origin idea is that this is task of checkSeparatorBetweenGroups.
Done.